src/ansys/dpf/composites/layup_info/material_properties.py

```diff
@@ -43,71 +43,69 @@
 class MaterialProperty(str, Enum):
     """Provides the material properties that are available."""
 
-    Engineering_Constants_E1: str = "young_modulus_x_direction"
```

Comment:
Following https://typing.python.org/en/latest/spec/enums.html#defining-members, in particular this section:

![image](https://github.com/user-attachments/assets/d07b0fc8-0330-4de5-9641-e19cf9ea63d7)